pbs_light.go

```diff
@@ -44,16 +44,18 @@ type AccountMetrics struct {
 	RequestMeter      metrics.Meter
 	BidsReceivedMeter metrics.Meter
 	PriceHistogram    metrics.Histogram
+	AdapterMetrics    map[string]*AdapterMetrics
```

Comment:
well the type here is a String, not a `Bidder`. it's a `bidder.BidderCode` but I don't think that work for a type?